apps/condo/domains/organization/schema/OrganizationEmployeeRequest.js

```diff
@@ -247,6 +269,18 @@ const OrganizationEmployeeRequest = new GQLListSchema('OrganizationEmployeeReque
                 }
             }
 
+            const userId = get(resolvedData, 'user')
+            if (userId) {
+                const user = await getByCondition('User', {
+                    id: userId,
+                    deletedAt: null,
+                })
+                if (user) {
+                    resolvedData['userName'] = user.name
```

Comment:
Just a food for thought, might be cool to double check names, maybe not here, but in User model

So that I cannot add links or something spammy

Reply:
That's a good point. I think it is relevant for all fields where the user can enter text